src/etc/inc/xmlparse.inc

```diff
@@ -248,7 +248,8 @@ function dump_xml_config_sub($arr, $indent) {
 						    (substr($ent, 0, 9) == "ldap_bind") ||
 						    (substr($ent, 0, 11) == "ldap_basedn") ||
 						    (substr($ent, 0, 18) == "ldap_authcn") ||
-						    (substr($ent, 0, 19) == "ldap_extended_query")) {
+						    (substr($ent, 0, 19) == "ldap_extended_query") ||
+							($ent == "customwidgettitle")) {
```

Comment:
Rather than adding a new key, why not use "descr" for this field name in the config, which already gets protected?